sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 - Added OS Patching Parameters to Compute Instance.
+- Change print behavior of various entity classes to match CLI `show` behavior.
```

Comment:
Please add the entry for this change for the upcoming SDK release. SDK 1.2.0 cannot have this change.